src/sagemaker/fw_utils.py

```diff
@@ -504,3 +505,16 @@ def python_deprecation_warning(framework, latest_supported_version):
     return PYTHON_2_DEPRECATION_WARNING.format(
         framework=framework, latest_supported_version=latest_supported_version
     )
+
+
+def _region_supports_debugger_feature(region_name):
```

Comment:
[nit] I think "feature" is redundant in both the code and prose

Reply:
+1. Removed "feature"